doc/src/manual/parallel-computing.md

```diff
@@ -299,11 +309,10 @@ julia> let B = B
 
 julia> @spawnat 2 whos();
 
-julia> 	From worker 2:	                             A    800 bytes  10×10 Array{Float64,2}
-	From worker 2:	                          Base               Module
-	From worker 2:	                          Core               Module
-	From worker 2:	                          Main               Module
-
+julia>  From worker 2:                               A    800 bytes  10×10 Array{Float64,2}
```

Comment:
may as well have the output look like what you'd get at the repl from running this, including strange spacing where that's just being faithful